tests/time_tests/src/timetests_helper/cli.h

```diff
@@ -26,6 +26,18 @@ static const char target_device_message[] =
     "plugin. "
     "The application looks for a suitable plugin for the specified device.";
 
+/// @brief message for vpu argument
+static const char performance_hint_message[] =
+    "Required. Enables performance hint for specified device.";
```

Comment:
This hint is not supported for GNA and Myriad. This tool is probably not for measuring their performance, but it is still nice to have functionality